lib/Drush/CommandFiles/core/DrupliconCommands.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace Drush\CommandFiles\core;
+
+class DrupliconCommands {
+
+  /*
+   * @todo not working until we modernize drush_get_global_options().
+   * @hook alter help
+   */
+  public function help($command) {
+    if ($command['command'] == 'global-options' && $command['#brief'] === FALSE) {
+      $command['options']['druplicon'] = [
+        'description' => 'Shows the druplicon as glorious ASCII art.',
+      ];
+    }
+  }
+
+  /**
+   * Print druplicon as post-command output.
+   *
+   * @hook post-command
```

Comment:
I just went to make a fix to the post-command hook.  The thing about the command hook is that, in AnnotatedCommand, this hook is called via the Symfony Console `Command` event.  This means that it takes different parameters than the other hooks -- namely, InputInterface and OutputInterface.  The advantage of doing things this way is that it's possible to hook 'native' Symfony commands in a mixed application.  Also, the Command Event happens before argument and option validation, so it's a good place to provide things like default options from configuration.

The odd thing about this is that post-command currently runs before the main command hook, as I alluded to above.  I was going to make this happen afterwards, but the different parameters for this event makes this weird.  I think that maybe the best thing to do is to rename the current 'command' hook to the 'commandevent' or 'symfonycommand' or 'symfony' hook, and make the 'command' hook behave as it does in Drush.